docs/orchestration/agents/overview.md

```diff
@@ -54,42 +54,17 @@ LocalAgent().start()
 
 The following configuration options are shared for all agents.
 
-### API Keys <Badge text="Cloud"/>
+### Authentication <Badge text="Cloud"/>
 
-Prefect agents rely on the use of a service account API key from Prefect Cloud. For
-information on API keys and how they are used visit the
-[API keys](../concepts/tokens.html) page.
+Prefect agents rely on the use of an API key from Prefect Cloud. For information on how to create and configure API keys, see the
+[API key documentation](../concepts/api_keys.md).
 
-When starting an Agent with Prefect Cloud, you'll need to provide the API key. There are a few different ways to do this:
-
-:::: tabs
-::: tab CLI
-
-```bash
-prefect agent <AGENT TYPE> start --token <SERVICE_ACCOUNT_API_KEY>
-```
-
-:::
-
-::: tab "Prefect Config"
-
-```toml
-# ~/.prefect/config.toml
```

Comment:
do environment variables or `config.toml` work for agents anymore? I don't see them anywhere in the updated docs

Reply:
They do -- they're just the same as normal auth so `PREFECT__CLOUD__API_KEY` and it's covered in the API key docs. I provide a link in the agent auth section

https://deploy-preview-4643--prefect-docs.netlify.app/orchestration/agents/overview.html#authentication

Reply:
(specifically for agents)

Reply:
You should set a separate auth key for agents via the environment anymore. The old environment variable will be passed through as a token to the client if set but if your client has an API key configured as well there will be a warning and the token will be ignored. If you do not have an API key configured globally and you set the old environment variable with either a token or API key, it will authenticate as you'd expect.

Reply:
ahhhh okay i see now we removed that logic